options/locale/locale_en-US.ini

```diff
@@ -680,6 +680,7 @@ block.list.none = You have not blocked any users.
 profile = Profile
 account = Account
 appearance = Appearance
+notifications = Notifications
```

Comment:
We already have one
![image](https://github.com/go-gitea/gitea/assets/18380374/fdc515d5-3677-4e9a-a485-12a7e6a678da)

Reply:
This is a history problem. Now we recommend to use existing translations if it is possible.